pkgs/applications/graphics/rawtherapee/default.nix

```diff
@@ -1,26 +1,32 @@
 { stdenv, fetchFromGitHub, pkgconfig, gtk, cmake, pixman, libpthreadstubs, gtkmm, libXau
-, libXdmcp, lcms2, libiptcdata, libcanberra, fftw, expat, pcre, libsigcxx 
+, libXdmcp, lcms2, libiptcdata, libcanberra, fftw, expat, pcre, libsigcxx
 , mercurial  # Not really needed for anything, but it fails if it does not find 'hg'
 }:
 
 stdenv.mkDerivation rec {
-  name = "rawtherapee-4.2";
-  
+  version = "4.2.1025";
+  name = "rawtherapee-" + version;
+
   src = fetchFromGitHub {
     owner = "Beep6581";
     repo = "RawTherapee";
-    rev = "4.2";
-    sha256 = "1v4px239vlmk9l8wbzlvlyni4ns12icxmgfz21m86jkd10pj5dgr";
+    rev = "dc4bbe906ba92ddc66f98a3c26ce19822bfb99ab";
+    sha256 = "0c5za9s8533fiyl32378dq9rgd5044xi8y0wm2gkr7krbdnx74l3";
   };
-  
+
   buildInputs = [ pkgconfig gtk cmake pixman libpthreadstubs gtkmm libXau libXdmcp
     lcms2 libiptcdata mercurial libcanberra fftw expat pcre libsigcxx ];
 
-  patchPhase = ''
-    patch -p1 < ${./sigc++_fix.patch}
-  '';
+  cmakeFlags = [
+    "-DCMAKE_BUILD_TYPE=release"
```

Comment:
I think `CMAKE_BUILD_TYPE=Release` is the default (`pkgs/development/tools/build-managers/cmake/setup-hook.sh:52`) and this is not case-sensitive, is it?